man/clrtrust.1.md

```diff
@@ -0,0 +1,141 @@
+Trusts can also be provided and modified by privileged users.
+
+## OPTIONS
+
+```
+Usage: clrtrust [-v|--verbose] [-h|--help] <command> [options]
+
+    Commands
+        generate    generates the trust store
+        list        list CAs
+        add         add trust to a CA
+        remove      remove trust to a CA
+        restore     restore trust to previously removed CA
+        check       sanity/consistency check of the trust store
+
+clrtrust <command> --help to get help on specific command.
+```
+
+Commands that modify the store require root privileges.
+
+
+## EXAMPLES
+
+### View the list of trusted CAs
+
+`clrtrust list`
+
+The `list` command has no arguments and outputs a list of certificates in the
+format below:
+
+```
+id: FA:B7:EE:36:97:26:62:FB:2D:B0:2A:F6:BF:03:FD:E8:7C:4B:2F:9B
+File: /var/cache/ca-certs/anchors/certSIGN_ROOT_CA.crt
+Authority: /C=RO/O=certSIGN/OU=certSIGN ROOT CA
+Expires: Jul  4 17:20:04 2031 GMT
+```
+
+- `id` uniquely identifies the certificate. It can be used as input to other
+  `clrtrust` commands such as  `clrtrust remove`.
+
+- `File`  contains the file path of the certificate in the trust store. The
+  certificate can be further inspected using the `openssl x509` command. For
+  example:
+
+  ```
+  openssl x509 -in /var/cache/ca-certs/anchors/certSIGN_ROOT_CA.crt -noout -text
+  ```
+
+- `Authority` shows the name of the organization that issued the certificate.
+  This field is extracted from the certificate file.
+
+- `Expires` shows the expiration date of the certificate. This field is
+  extracted from the certificate file.
+
+
+### Add (trust) a root CA
+
+`clrtust add` 
+
+Adding a root CA means that the operating systems will trust the root CA,
+certificate chains issued by the authority will be accepted as valid, and if an
+application validates the peer, a connection will be established.
+
+The `add` command takes one or more certificate files as arguments. The
+certificate file must be PEM-encoded, one certificate per file. For example,
+assuming the CA certificate is in the file `~/PrivateCA.pem` the following
+command will add it to the store:
+
+```
+clrtrust add ~/PrivateCA.pem
+```
+
+If the certificate file is in a different format, use `openssl x509`
+command to convert to PEM. For example:
+
+```
+openssl x509 -in PrivateCA.cer -inform der -out PrivateCA.pem -outform pem
+```
+
+### Remove (distrust) a Root CA
+
+`clrtrust remove`
+
+Removing (distrusting) a root CA means that the operating systems will no
+longer trust the root CA, certificate chains issued by the authority will no
+longer be accepted as valid, and if an application validates the peer, a
+connection will not be established.
+
+The `remove` command takes one argument to identify which trust to remove. The
+argument can be an `id` of the certificate (see the `list` command) or the file
+path of the certificate.
+
+For example, to distrust the certificate added in the previous example:
+
```

Comment:
You've described `add` and `remove` and `list`, but still missing is restore/generate and check.